docs/appkit/shared/smart-accounts.mdx

```diff
@@ -57,14 +57,26 @@ New AppKit Universal Wallet Users will be given smart accounts by default when t
 
 There is a small additional cost for activating your smart account. The activation fee is added to the first transaction and covers the network fees required for deploying the new smart contract onchain.
 
-# What are account names?
+### Can I export my Smart Account?
+
+No, you cannot export your Smart Account. The Smart Account (SA) is deployed by the EOA and is owned by the EOA. Your EOA account will always be exportable.
+Also is good to know that SA don't have seedphrases.
+
+### Can I rotate the signer ?
+
+Yes, you can rotate the signer manage it elsewhere if they want too. We expect there to be more UIs for this wallet in the future including Safe-Wallet
+
+### Can I withdraw all my funds from my Smart Account?
+
+Yes, you can withdraw all your funds from your Smart Account.
+
+### What are account names?
```

Comment:
isn't all of this repeated above?

Reply:
yes , it's two time because of our beautiful docs setup.